### ISSUE TITLE
Import of zarr into OMERO merge-ci latest-ci

### ISSUE THREAD

**Issue by pwalczysko** [ome/ZarrReader#39]
On both merge-ci (on idr2-slot3) and latest-ci (on idr1-slot2), the import of ome.zarr files fails with the error below.
The files were imported previously into OMERO successfully using the same servers and envs.

```
export OMERODIR=/home/omero/workspace/OMERO-server/OMERO.server
source /home/omero/workspace/OMERO-server/.venv3/bin/activate
(.venv3) bash-4.2$ pwd
/uod/idr-scratch/petr-test/sub-resolutions
(.venv3) bash-4.2$ omero import --transfer=ln_s --skip=checksum --parallel-upload=10 -T Dataset:name:test-zarr LuCa-7color_Scan1.ome.zarr/ --depth=10
```

causes the following error reports (the error repeats several times, probably extra error for each subfolder of the ome.zarr)

<details>

```
2022-08-05 09:07:17,137 40610      [      main] ERROR     ome.formats.importer.cli.ErrorHandler - FILE_EXCEPTION: /uod/idr-scratch/petr-test/sub-resolutions/LuCa-7color_Scan1.ome.zarr/0/2/0/3/0/4/0
java.lang.Exception: java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonKey
	at ome.formats.importer.ImportCandidates.singleFile(ImportCandidates.java:469)
	at ome.formats.importer.ImportCandidates.handleFile(ImportCandidates.java:576)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:387)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:384)
	at org.apache.commons.io.DirectoryWalker.walk(DirectoryWalker.java:355)
	at ome.formats.importer.ImportCandidates.execute(ImportCandidates.java:381)
	at ome.formats.importer.ImportCandidates.<init>(ImportCandidates.java:222)
	at ome.formats.importer.ImportCandidates.<init>(ImportCandidates.java:174)
	at ome.formats.importer.cli.CommandLineImporter.<init>(CommandLineImporter.java:148)
	at ome.formats.importer.cli.CommandLineImporter.main(CommandLineImporter.java:997)
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonKey
	at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.hasAsKey(JacksonAnnotationIntrospector.java:1080)
	at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector._addFields(POJOPropertiesCollector.java:495)
	at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector.collectAll(POJOPropertiesCollector.java:421)
	at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector.getPropertyMap(POJOPropertiesCollector.java:386)
	at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector.getProperties(POJOPropertiesCollector.java:233)
	at com.fasterxml.jackson.databind.introspect.BasicBeanDescription._properties(BasicBeanDescription.java:164)
	at com.fasterxml.jackson.databind.introspect.BasicBeanDescription.findProperties(BasicBeanDescription.java:239)
	at com.fasterxml.jackson.databind.deser.BasicDeserializerFactory._findCreatorsFromProperties(BasicDeserializerFactory.java:328)
	at com.fasterxml.jackson.databind.deser.BasicDeserializerFactory._constructDefaultValueInstantiator(BasicDeserializerFactory.java:272)
	at com.fasterxml.jackson.databind.deser.BasicDeserializerFactory.findValueInstantiator(BasicDeserializerFactory.java:223)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerFactory.buildBeanDeserializer(BeanDeserializerFactory.java:261)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerFactory.createBeanDeserializer(BeanDeserializerFactory.java:150)
	at com.fasterxml.jackson.databind.deser.DeserializerCache._createDeserializer2(DeserializerCache.java:414)
	at com.fasterxml.jackson.databind.deser.DeserializerCache._createDeserializer(DeserializerCache.java:349)
	at com.fasterxml.jackson.databind.deser.DeserializerCache._createAndCache2(DeserializerCache.java:264)
	at com.fasterxml.jackson.databind.deser.DeserializerCache._createAndCacheValueDeserializer(DeserializerCache.java:244)
	at com.fasterxml.jackson.databind.deser.DeserializerCache.findValueDeserializer(DeserializerCache.java:142)
	at com.fasterxml.jackson.databind.DeserializationContext.findRootValueDeserializer(DeserializationContext.java:591)
	at com.fasterxml.jackson.databind.ObjectMapper._findRootDeserializer(ObjectMapper.java:4733)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4594)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3561)
	at com.bc.zarr.ZarrUtils.fromJson(ZarrUtils.java:113)
	at com.bc.zarr.ZarrGroup.ensureZarrFormatIs2(ZarrGroup.java:120)
	at com.bc.zarr.ZarrGroup.validateGroupToBeOpened(ZarrGroup.java:111)
	at com.bc.zarr.ZarrGroup.open(ZarrGroup.java:102)
	at com.bc.zarr.ZarrGroup.open(ZarrGroup.java:95)
	at com.bc.zarr.ZarrGroup.open(ZarrGroup.java:87)
	at loci.formats.services.JZarrServiceImpl.getGroupAttr(JZarrServiceImpl.java:104)
	at loci.formats.in.ZarrReader.initFile(ZarrReader.java:164)
	at loci.formats.FormatReader.setId(FormatReader.java:1443)
	at loci.formats.ImageReader.setId(ImageReader.java:849)
	at ome.formats.importer.OMEROWrapper$4.setId(OMEROWrapper.java:167)
	at loci.formats.ReaderWrapper.setId(ReaderWrapper.java:650)
	at loci.formats.ChannelFiller.setId(ChannelFiller.java:234)
	at loci.formats.ReaderWrapper.setId(ReaderWrapper.java:650)
	at loci.formats.ChannelSeparator.setId(ChannelSeparator.java:293)
	at loci.formats.ReaderWrapper.setId(ReaderWrapper.java:650)
	at loci.formats.Memoizer.setId(Memoizer.java:662)
	at loci.formats.ReaderWrapper.setId(ReaderWrapper.java:650)
	at ome.formats.importer.ImportCandidates.singleFile(ImportCandidates.java:427)
	... 14 common frames omitted
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.annotation.JsonKey
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 54 common frames omitted
```

</details>

cc @joshmoore @sbesson @dgault

**Comment by joshmoore** [comment]
When did it start, @pwalczysko? Do all files fail now?

**Comment by pwalczysko** [comment]
> When did it start, @pwalczysko?

Detected during last week, say, Thursday or Wednesday. Not sure when did it start, could have started during my AL or even before, even June 2022 would be plausible.

> Do all files fail now?

Yes, all ``ome.zarr`` files I tried, not only the ones I have created with bf2raw, but also the ones from the history of the bash on merge-ci, the same error. The error shows itself also with ``import -f``.
